scripts/stack/run-stack.sh

```diff
@@ -58,6 +58,7 @@ fi
 if [ "${no_connectors:-}" == false ]; then
   echo "Starting Elastic Connectors..."
 
+  source ./copy-config.sh
```

Comment:
This is done a few lines up if `$run_configurator == "yes"`. What if instead we pull that out of the `if` block, and move this `source ./copy-config.sh` call up to line 41? 

Also, I think we should edit `copy-config.sh` to make a call to `make config` to ensure that `config.yml` exists.

Reply:
> move this `source ./copy-config.sh` call up to line 41?

Good idea


> Also, I think we should edit `copy-config.sh` to make a call to `make config` to ensure that `config.yml` exists.

Great suggestion, will update.

Reply:
Updated in https://github.com/elastic/connectors/pull/2445/commits/4d6d88a4e34455b9171d0efca31d52833422bf8a